doc/userguide/install.rst

```diff
@@ -90,18 +90,20 @@ Rust support::
 Ubuntu/Debian
 """""""""""""
 
+.. Important:: The following instructions require `sudo` be installed.
+
 Minimal::
 
     # Installed Rust and cargo as indicated above
-    apt-get install build-essential git libjansson-dev libpcap-dev \
+    sudo apt-get install build-essential git libjansson-dev libpcap-dev \
```

Comment:
I think from what I understand from the last PR, Jason suggested that we add the note that sudo is required (like you did) but,
1. I'm not sure if it is supposed to go as "Important", is there no other note variant offered by Sphinx?
2. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.

Reply:
> I think from what I understand from the last PR, Jason suggested that we add the note that sudo is required (like you did) but,
> 1. I'm not sure if it is supposed to go as "Important", is there no other note variant offered by Sphinx?
> 2. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.

1. Based on the definition I saw of the note variants, the use of `Important` was to let the users know that the provided information was to he followed for their work to be successful, that's why I used `Important`.

2. So I should remove all the sudos I added, and just leave the `Important` alerts then?

Reply:
@inashivb

1. The only thing I wasn't able to do was the author line though; is there something else I'm missing?

Reply:
> @inashivb
> 
>     1. The only thing I wasn't able to do was the author line though; is there something else I'm missing?

Also, the commit message length. Thanks :)

Reply:
> > I think from what I understand from the last PR, Jason suggested that we add the note that sudo is required (like you did) but,
> > 
> > 1. I'm not sure if it is supposed to go as "Important", is there no other note variant offered by Sphinx?
> > 2. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.
> 
>     1. Based on the definition I saw of the note variants, the use of `Important` was to let the users know that the provided information was to he followed for their work to be successful, that's why I used `Important`.
> 
>     2. So I should remove all the sudos I added, and just leave the `Important` alerts then?

hmm.. I'll let @jasonish comment on this just to be sure.

Reply:
> > @inashivb
> > 
> >     1. The only thing I wasn't able to do was the author line though; is there something else I'm missing?
> 
> Also, the commit message length. Thanks :)

So I should reduce the commit message length, OK.

Please advise me on how to add that author line, I can't find any commits I can use as a sample on how to go about it.

Reply:
> > > I think from what I understand from the last PR, Jason suggested that we add the note that sudo is required (like you did) but,
> > > 
> > > 1. I'm not sure if it is supposed to go as "Important", is there no other note variant offered by Sphinx?
> > > 2. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.
> > 
> >     1. Based on the definition I saw of the note variants, the use of `Important` was to let the users know that the provided information was to he followed for their work to be successful, that's why I used `Important`.
> > 
> >     2. So I should remove all the sudos I added, and just leave the `Important` alerts then?
> 
> hmm.. I'll let @jasonish comment on this just to be sure.

OK then.

Thanks.

Reply:
> Please advise me on how to add that author line, I can't find any commits I can use as a sample on how to go about it.

```
git config user.name First Last
git config user.mail myemail@example.com
git commit --amend --author="First Name <firstnamelastname@email.org>"
```
will fix the email of the most recent commit.

You'll also want to reword to adjust the first line down to our format.

Reply:
The first line of the commit message, right?

Please help me with a sample, I don't seem to be getting it right.

Reply:
> 2\. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.

I guess we're a little undecided here, and the ticket didn't clarify it.  However, for install instructions I think it might make sense to have the sudo..  Because you don't want to suggest switching to root, as when it comes to build it could be forgotten to switch back to a non-root user, and you don't want to be building as root.

Its also easier for a container user to just add sudo, then a normal user having to manually add sudo to the beginning of each command.

Reply:
> > 2\. The commands should stay w/o sudo to make it a copy-paste job if one were to put these in a container or other env missing sudo.
> 
> I guess we're a little undecided here, and the ticket didn't clarify it.  However, for install instructions I think it might make sense to have the sudo..  Because you don't want to suggest switching to root, as when it comes to build it could be forgotten to switch back to a non-root user, and you don't want to be building as root.
> 
> Its also easier for a container user to just add sudo, then a normal user having to manually add sudo to the beginning of each command.

One of the instructionsn; re: backports, suggested switching to `root`, so I didn't add `sudo` there.

Reply:
Yes! Thanks you very much.